test/browser/custom-elements.js

```diff
@@ -84,6 +84,93 @@ describe('Custom Elements', function () {
     domUI.dispose();
   });
 
+  it('should dispose vtree$ after the custom element has been destroyed', function () {
+    let log = [];
+    let number$ = Rx.Observable.range(1, 2).controlled();
```

Comment:
'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).